user-group/confirmed-change.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
```

Comment:
On smaller screens, the footer logo is attached to the content

<img width="652" alt="screen shot 2017-02-07 at 11 47 10" src="https://cloud.githubusercontent.com/assets/109850/22688137/34938e26-ed2b-11e6-967b-929d2d66cffe.png">